org/contributors.yml

```diff
@@ -538,6 +538,7 @@ contributors:
 - sebGilR
 - selzoc
 - services-api-ci
+- servicesenablement
```

Comment:
A bot should not be listed as contributor. Org automation is supposed to add the bots as github org member.

Reply:
Done!